packages/ProgressAccordion/README.md

```diff
@@ -0,0 +1,53 @@
+## ProgressAccordion
+
+The `<ProgressAccordion>` component displays questionnaire responses in a collapsible accordion format.
+
+### Installation
+
+`> npm install --save @paprika/progress-accordion`
+or
+`> yarn add @paprika/progress-accordion`
+
+### Usage
+
+```js
+import ProgressAccordion from "@paprika/progress-accordion";
+
+const responses = [
+  {
+    id: "1",
+    heading: "Who is on first?",
+    body: <p>Yes.</p>,
+  },
+];
+
+<ProgressAccordion activeIndex={1} activeStatus="5 days idle" a11yText="Responses">
+  <ProgressAccordion.Item id="0" label="Zero">
+    <ProgressAccordion.Responses responses={responses} />
```

Comment:
We haven't used an object into our component, this should be a component itself 

`<Response id="" heading="">[node]<Response>`

Then you could abstract the content in an array.